commit 7d472367e2a5edd1628339181e3689b358761316
Merge: d841028 782464e
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Fri Mar 27 16:14:55 2015 -0700

    Merge pull request #1713 from epatel/Development
    
    Parentheses need here otherwise sanitycheck.h does wrong thing on line 222

diff --git a/Marlin/Conditionals.h b/Marlin/Conditionals.h
index c471ec4..c72f9d1 100644
--- a/Marlin/Conditionals.h
+++ b/Marlin/Conditionals.h
@@ -362,7 +362,7 @@
   #define HAS_AUTO_FAN_1 (defined(EXTRUDER_1_AUTO_FAN_PIN) && EXTRUDER_1_AUTO_FAN_PIN >= 0)
   #define HAS_AUTO_FAN_2 (defined(EXTRUDER_2_AUTO_FAN_PIN) && EXTRUDER_2_AUTO_FAN_PIN >= 0)
   #define HAS_AUTO_FAN_3 (defined(EXTRUDER_3_AUTO_FAN_PIN) && EXTRUDER_3_AUTO_FAN_PIN >= 0)
-  #define HAS_AUTO_FAN HAS_AUTO_FAN_0 || HAS_AUTO_FAN_1 || HAS_AUTO_FAN_2 || HAS_AUTO_FAN_3
+  #define HAS_AUTO_FAN (HAS_AUTO_FAN_0 || HAS_AUTO_FAN_1 || HAS_AUTO_FAN_2 || HAS_AUTO_FAN_3)
   #define HAS_FAN (defined(FAN_PIN) && FAN_PIN >= 0)
 
   /**
